dynatrace/environment_v2/settings.py

```diff
@@ -0,0 +1,97 @@
+        """Lists settings
+
+        :return: a list of settings with details
+        """
+        params = {
+            "schemaIds": schema_id,
+            "scope": scope,
+            "fields": fields,
+            "externalIds": external_ids,
+            "filter": filter,
+            "sort": sort,
+            "pageSize": page_size,
+        }
+        return PaginatedList(Settings, self.__http_client, target_url=self.ENDPOINT, list_item="items", target_params=params)
+    
+    def post(self,external_id,object_id,schema_id,schema_version,scope, value,validate_only):
+        """Creates a new settings object
+
+        :param external_id: External identifier for the object being created
+        :param object_id: The ID of the settings object that should be replaced. Only applicable if an external identifier
+        :param object_id: the ID of the object
+        :param schema_id: The schema on which the object is based
+        :param schema_version: The version of the schema on which the object is based.
+        :param scope The scope that the object targets. For more details, please see Dynatrace Documentation﻿.
+        :param value The value of the setting.
+        :return: a Settings object
+        """        
+        params = {
+            "validate_only": validate_only,
+        }
+        body =[ {
+            "externalId" : external_id,
+            "objectId": object_id,
+            "schemaId": schema_id,
+            "schemaVersion": schema_version,
+            "scope": scope,
+            "value" : value
+
+        }]
+   
+        response = self.__http_client.make_request(self.ENDPOINT,params=body, method="POST",query_params=params).json()
+        return response
+    
+    
+    def get(self, object_id: str):
+        """Gets parameters of specified settings object
+
+        :param object_id: the ID of the object
+        :return: a Settings object
+        """
+        response = self.__http_client.make_request(f"{self.ENDPOINT}/{object_id}").json()
+        return Settings(raw_element=response)
+
+    def update(self, object_id: str,  value):
+        """Updates an existing settings object
+        :param object_id: the ID of the object
+
+        """
+        return self.__http_client.make_request(path=f"{self.ENDPOINT}/{object_id}", params=value, method="PUT")
+
+    def delete(self, object_id: str):
+        """Deletes the specified object
+
+        :param object_id: the ID of the object
+        :return: HTTP response
+        """
+        return self.__http_client.make_request(path=f"{self.ENDPOINT}/{object_id}", method="DELETE")
+
+class Settings(DynatraceObject):
+    value = None
+    def _create_from_raw_data(self, raw_element: Dict[str, Any]):
+        # Mandatory
+        self.objectId: str = raw_element["objectId"]
+        self.value: str = raw_element["value"]
+    def to_json(self):
```

Comment:
I understand the motivation for this but I don't think we'd include this as all it is is a 'getter' that returns the value attribute. If this is needed they can just access the 'value' attribute of the settings object. We don't have anything like this for other objects. to_json() could also confuse someone if they expected the _whole_ json including the object id.

Reply:
removed to_json